modules/calendar/calendar_embedder.py

```diff
@@ -55,8 +54,13 @@ async def embed_event_pages(
 					page_num=page_num,
 					max_results=results_per_page,
 				)
-				sender = interaction.send if interaction.message is None else interaction.message.edit
-				await sender(embed=embed)
+				sender = (
+					interaction.send
+					if interaction.message is None
+					else interaction.message.edit
+				)
+				message = await sender(embed=embed)  # type: ignore
```

Comment:
interaction.send won't necessarily return a message (unless defer was used beforehand)

To get the message, a separate call using `await interaction.original_message()` needs to be used.

It would be preferable to use `interaction.edit_original_message` and `interaction.delete_original_message` where possible, but for things like adding reactions, that can't be done. (You can add Buttons with edit_original_message though)